gridsome.config.js

```diff
@@ -57,6 +57,22 @@ module.exports = {
           },
         ],
       },
+      {
+        name: "troubleshooting",
+        sections: [
+          {
+            title: "Troubleshooting your solutions",
+            items: [
+              "/troubleshooting-solutions/",
+              "/troubleshooting-solutions/1-almost-good-but/",
+              "/troubleshooting-solutions/2-getting-information/",
+              "/troubleshooting-solutions/3-something-crashes/",
+              "/troubleshooting-solutions/4-miscellaneous/",
+              "/troubleshooting-solutions/appendices/",
```

Comment:
These are not working because the paths are wrong. Remember the path is the file path without `content` and `.md`.

So `content/recipes/troubleshooting-solutions/1-almost-good-but.md` is at `/recipes/troubleshooting-solutions/1-almost-good-but/`.

Reply:
x/ damn it... x/

updating

Reply:
meh... I need to sleep... x(

Still not showing up. I'll try once more time, otherwise that will be for tomorrow